CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 2024-10-22
```

Comment:
I will describe the release schedule in this PR: https://github.com/jupyter/docker-stacks/pull/2072